.github/workflows/docbuild.yml

```
@@ -54,7 +54,6 @@ jobs:
       - name: Install Python dependencies
         working-directory: ncs
         run: |
-          sudo pip3 install -U setuptools wheel pip
           pip3 install -r nrf/doc/requirements.txt
```

can you also switch to `pip`
```suggestion
          pip install -r nrf/doc/requirements.txt
```